src/modules/system-settings/application/dto/system-settings-response.dto.ts

```
@@ -0,0 +1,85 @@
+}
+
+export class ExportSettingsResponseDto {
+  @ApiProperty()
+  version: string;
+
+  @ApiProperty()
+  exportedAt: Date;
+
+  @ApiProperty()
+  settings: SystemSettingsResponseDto;
+}
```

ESLint

[error] Insert `⏎`

(prettier/prettier)